boinc-client/start-utils.sh

```diff
@@ -0,0 +1,26 @@
+get_int_xml_val() {
+  local xml_val=$(xml_grep --text_only "$1" "$2")
+  local xml_int_val=$(awk "BEGIN {print int($xml_val)}")
+  echo $xml_int_val
+}
+
+update_float_xml_val_with_int() {
+  sed -i -e "s|<$1>[0-9a-z.]\{1,\}</$1>|<$1>$2.000000</$1>|g" "$3"
+}
+
+validate_ram_settings() {
+  local cfg_ram_max_busy=$(get_int_xml_val "$cfg_ram_max_busy_xml_key" "$prefs_file_path")
+  local cfg_ram_max_idle=$(get_int_xml_val "$cfg_ram_max_busy_xml_key" "$prefs_file_path")
+
+  echo "Validating boinc RAM settings"
+
+  if [[ ! -z $cfg_ram_max_busy && $cfg_ram_max_busy -gt $treshold_ram_settings_pct ]]; then
```

Comment:
Being nitpicky, but just wanted to correct spelling here to save any possible mistakes in the future.

```suggestion
  if [[ ! -z $cfg_ram_max_busy && $cfg_ram_max_busy -gt $threshold_ram_settings_pct ]]; then
```

Reply:
Absolutely :) Thanks